cmd/maya-apiserver/app/server/volume_endpoint_v1alpha1.go

```diff
@@ -91,6 +95,16 @@ func (v *volumeAPIOpsV1alpha1) httpGet() (interface{}, error) {
 	return v.read(volName)
 }
 
+//httpPut deals with http PUT request
+func (v *volumeAPIOpsV1alpha1) httpPut() (interface{}, error) {
+	path := strings.TrimSpace(strings.TrimPrefix(v.req.URL.Path, "/latest/volumes"))
+	if strings.Contains(path, "/resize/") {
+		return v.resize()
+	} else {
```

Comment:
No need for else here. The usual Go style for that is

if strings.Contains(path, "/resize/") {
		return v.resize()
	}
	return nil, CodedError(405, fmt.Sprintf("failed to process a PUT request"))
	}